packages/marshal/src/marshal.js

```diff
@@ -382,7 +393,7 @@ export function makeMarshal(
           }
           case 'symbol': {
             const { name } = rawTree;
-            return passableSymbolForName(name);
+            return passableSymbolForName(`${name}`);
```

Comment:
Is this just a drive by improvement (thanks!), or did you notice this because of the enhanced type checking? Do you understand why?

Reply:
@michaelfig what's the reason for this change?

Reply:
It's a cast needed by the newer typescript: `name` could be a `number` or something like that.

Reply:
Ha I think I fixed the root cause of that with a different cast. Will see if I can undo this one

Reply:
I indeed did, reverted this change.